src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -71,32 +83,56 @@ end
 
 # this method is used when the limiter is constructed as for shock-capturing volume integrals
 function SubcellLimiterIDP(equations::AbstractEquations, basis;
-                           local_minmax_variables_cons = String[],
+                           local_twosided_variables_cons = String[],
                            positivity_variables_cons = String[],
                            positivity_variables_nonlinear = [],
                            positivity_correction_factor = 0.1,
+                           local_onesided_variables_nonlinear = [],
                            max_iterations_newton = 10,
                            newton_tolerances = (1.0e-12, 1.0e-14),
                            gamma_constant_newton = 2 * ndims(equations))
-    local_minmax = (length(local_minmax_variables_cons) > 0)
+    local_twosided = (length(local_twosided_variables_cons) > 0)
+    local_onesided = (length(local_onesided_variables_nonlinear) > 0)
     positivity = (length(positivity_variables_cons) +
                   length(positivity_variables_nonlinear) > 0)
 
-    local_minmax_variables_cons_ = get_variable_index.(local_minmax_variables_cons,
-                                                       equations)
+    # When passing `min` or `max` in the elixir, the specific function of Base is used.
+    # To speed up the simulation, we replace it with `Trixi.min` and `Trixi.max` respectively.
+    local_onesided_variables_nonlinear_ = Tuple{Function, Function}[]
+    for (variable, min_or_max) in local_onesided_variables_nonlinear
+        if min_or_max === Base.max
+            push!(local_onesided_variables_nonlinear_, tuple(variable, max))
+        elseif min_or_max === Base.min
+            push!(local_onesided_variables_nonlinear_, tuple(variable, min))
+        elseif min_or_max === Trixi.max || min_or_max === Trixi.min
+            push!(local_onesided_variables_nonlinear_, tuple(variable, min_or_max))
```

Comment:
```suggestion
            push!(local_onesided_variables_nonlinear_, (variable, max))
        elseif min_or_max === Base.min
            push!(local_onesided_variables_nonlinear_, (variable, min))
        elseif min_or_max === Trixi.max || min_or_max === Trixi.min
            push!(local_onesided_variables_nonlinear_, (variable, min_or_max))
```
Don't think `tuple` is necessary here, is it?

Reply:
Done in [62946c2](https://github.com/trixi-framework/Trixi.jl/pull/1792/commits/62946c2220b1005ac1ff9e8976425590d22e27e9)